### PR TITLE
Update 01-setup.md

### PR DESCRIPTION
**Pull request by phashish**
Fixed the authentication doc from

| **userCanEditFullName** | True if users can view their user name. | true |

to 

| **userCanEditFullName** | True if users can edit their full name. | true |

### CODE REVIEW

File: versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ void run(List<String> args) async {
 | **emailSignInFailureResetTime** | The reset period for email sign in attempts. Defaults to 5 minutes.| 5min |
 | **userCanEditUserImage** | True if users can update their profile images.| true |
 | **userCanEditUserName** | True if users can edit their user names. | true |
-| **userCanEditFullName** | True if users can view their user name. | true |
```
Comment (SandPod) [inline versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md:100]: Suggestion: Apply this change to the non versioned docs as well.

If you search for this line you can modify it in all places.

**Review by SandPod — CHANGES_REQUESTED** [review]
Thank you for fixing this up! 🙏 

Could you please apply this fix to all versions of the documentation?
It would unfortunately be reverted once we release a new version of our documentation the way it is applied now.

File: versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md (hunk 1 above)
Comment (phashish) [reply versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md:100]: Sure, on it.

Comment (phashish) [reply versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md:100]: https://github.com/serverpod/serverpod_docs/pull/169

Comment (SandPod) [reply versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md:100]: Thanks for being so quick on this. Could you add that change as a commit to this PR instead? Then we have the full fix as one change in main later.

Comment (phashish) [reply versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md:100]: Done. Please check.

**Review by SandPod — APPROVED** [review]
LGTM! 👍

**Pull request merged**